papers/Mathematics/paper.Rmd

```diff
@@ -0,0 +1,269 @@
+
+## Univariate Outliers
+
+Researchers frequently attempt to identify outliers using measures of deviation from the center of a variable's distribution. One of the most popular such procedure is the _z_ score transformation, which computes the distance in standard deviation (SD) from the mean. However, as mentioned earlier, this popular method is not robust. Therefore, for univariate outliers, it is recommended to use the median along with the Median Absolute Deviation (MAD), which are more robust than the interquartile range or the mean and its standard deviation [@leys2019outliers; @leys2013outliers].
+
+Researchers can identify outliers based on robust (i.e., MAD-based) _z_ scores using the `check_outliers()` function of the  *{performance}* package, by specifying `method = "zscore_robust"`. Although @leys2013outliers suggest a default threshold of 2.5 and @leys2019outliers a threshold of 3, *{performance}* uses by default a less conservative threshold of ~3.29.^[3.29 is an approximation of the two-tailed critical value for _p_ < .001, obtained through `qnorm(p = 1 - 0.001 / 2)`. We chose this threshold for consistency with the thresholds of all our other methods.] That is, data points will be flagged as outliers if they go beyond +/- ~3.29 MAD. Users can adjust this threshold using the `threshold` argument, as demonstrated below.
+
+```{r z_score}
+library(performance)
+
+# Create some artificial outliers and an ID column
+data <- rbind(mtcars[1:4], 42, 55)
+data <- cbind(car = row.names(data), data)
+
+outliers <- check_outliers(data, method = "zscore_robust", ID = "car")
+outliers
+```
+
+The row numbers of the detected outliers can be obtained by using `which()` on the output object, which can be used for exclusions for example:
+
+```{r}
+which(outliers)
+
+data_clean <- data[-which(outliers), ]
+```
+
+All `check_outliers()` output objects possess a `plot()` method, meaning it is also possible to visualize the outliers:
+
+```{r univariate, fig.cap = "Visual depiction of outliers using the robust z-score method."}
+library(see)
+
+plot(outliers)
+```
+
+Other univariate methods are available, such as using the interquartile range (IQR), or based on different intervals, such as the Highest Density Interval (HDI) or the Bias Corrected and Accelerated Interval (BCI). These methods are documented and described in the function's [help page](<https://easystats.github.io/performance/reference/check_outliers.html>).
```

Comment:
I have no idea about those, that's why I try to stick to recommandations 😅. @DominiqueMakowski added this part. Dom could you explain why/when it might be useful to use an interval instead of, say, MAD or IQR?

Reply:
Bump @DominiqueMakowski